apps/aeutil/lib/parser.ex

```diff
@@ -5,9 +5,9 @@ defmodule Aeutil.Parser do
 
   def to_atom!(key) when is_atom(key), do: key
   def to_atom!(key) when is_binary(key), do: String.to_atom(key)
-  def to_atom!(_key), do: throw("Key is neither atom nor string")
+  def to_atom!(key), do: key
```

Comment:
This is a behavior change and I'm not sure how this will effect our code. If we want this behaviour change you shold remove the `!` in function name.

Reply:
This is used only in the `Serialization` module for converting string map keys to atom keys. I will remove the `!` though.